packages/core/src/modules/dcql/models/DcqlCredentialsForRequest.ts

```diff
@@ -0,0 +1,20 @@
+import type { JsonObject } from '../../../types'
+import type { MdocRecord } from '../../mdoc'
+import type { SdJwtVcRecord } from '../../sd-jwt-vc'
+import type { W3cCredentialRecord } from '../../vc'
+import type { MdocNameSpaces } from '@animo-id/mdoc'
+
+/**
+ * Mapping of credential query IDs to the selected credential record and the disclosed payload.
+ */
+export type DcqlCredentialsForRequest = Record<
+  string,
+  | {
+      credentialRecord: MdocRecord
```

Comment:
Should we add format keys here as well to make it easy to differentiate based on format? 

So format: ClaimFormat.MsoMdoc